### PR TITLE
Allow duplicate on groups

### LINKED ISSUE #1106: Duplicate for groups
**Is your feature request related to a problem? Please describe.**
Duplicate auras is the only (hidden) option to un-link auras from a previous group
But duplicate is only available for auras

**Describe the solution you'd like**
Make duplicate handle groups too

### PR DESCRIPTION
**Pull request by mrbuds**
# Description
Show "Duplicate" option on groups

Fixes #1106

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?
- [x] duplicate same group of 3 auras 3 times

### CODE REVIEW

**Comment by Stanzilla** [comment]
this is your friendly neighbourhood minor bump reminder

**Comment by mrbuds** [comment]
wooops done

File: WeakAurasOptions/WeakAurasOptions.lua [hunk 1]
```diff
@@ -72,64 +72,72 @@ function WeakAuras.MultipleDisplayTooltipDesc()
...
+  WeakAuras.Add(newData)
+  WeakAuras.NewDisplayButton(newData)
+  if(newParent or data.parent) then
+    local parentId = newParent or data.parent
+    local parentData = WeakAuras.GetData(parentId)
+    local index
+    if newParent then
+      index = #parentData.controlledChildren + 1
+    else
+      for i, childId in pairs(parentData.controlledChildren) do
```
Comment (InfusOnWoW) [inline WeakAurasOptions/WeakAurasOptions.lua]: There's a utility function in Utils.lua, tIndexOf that does this, so you can replace this loop with:
index = tIndexOf(parentData.controlledChildren, data.id)

File: WeakAurasOptions/WeakAurasOptions.lua [hunk 2]
```diff
@@ -72,64 +72,72 @@ function WeakAuras.MultipleDisplayTooltipDesc()
...
+    newData.controlledChildren = {}
+  end
+  WeakAuras.Add(newData)
+  WeakAuras.NewDisplayButton(newData)
+  if(newParent or data.parent) then
+    local parentId = newParent or data.parent
+    local parentData = WeakAuras.GetData(parentId)
+    local index
+    if newParent then
+      index = #parentData.controlledChildren + 1
```
Comment (InfusOnWoW) [inline WeakAurasOptions/WeakAurasOptions.lua]: If you intialize index to #parentData.controlledChildren here...

File: WeakAurasOptions/WeakAurasOptions.lua [hunk 3]
```diff
@@ -72,64 +72,72 @@ function WeakAuras.MultipleDisplayTooltipDesc()
...
+      for i, childId in pairs(parentData.controlledChildren) do
+        if(childId == data.id) then
+          index = i
+          break
+        end
       end
     end
     if(index) then
-      local newIndex = index + 1;
+      local newIndex = index + 1
```
Comment (InfusOnWoW) [inline WeakAurasOptions/WeakAurasOptions.lua]: then you can replace this block with just tinsert(parentData.controlledChildren, index + 1, newData.id)

**Pull request merged**